docs/changelog-configuration.md

```diff
@@ -11,6 +11,13 @@ This document describes all settings that are supported in
 
 ## General options
 
+### `add_plugin_period` (boolean)
+
+The default value is `false`.
```

Comment:
Further below we're using the following wording for options whose default for newly created configs differs from existing configs:
```suggestion
The default value is `false` for existing configurations, and `true` for
new configurations.
```